src/main/resources/entity-types/openaccess/drv_openaccess_party.json5

```diff
@@ -0,0 +1,57 @@
+{
+  id: '0f8f6d6f-504f-4aa9-9d98-9779aaaa907b',
+  name: 'drv_openaccess_party',
+  private: false,
+  fromClause: 'src_oa_party AS party\nLEFT JOIN src_oa_alternate_email_address ON alternate_email_address.aea_owner_fk=party.p_id\nLEFT JOIN src_oa_refdata_value AS p_institution_level_1 ON p_institution_level_1.rdv_id=party.p_institution_level_1_fk',
+  columns: [
+    {
+      name: 'party_id',
+      dataType: {
+        dataType: 'openUUIDType',
+      },
+      queryable: true,
+      visibleByDefault: true,
+      valueGetter: 'party.p_id',
+    },
```

Comment:
I believe at least 1 column needs `isIdColumn: true` (something like that... I might have the property name wrong).

Reply:
I think I did tick the box in entity type creator, but it wasn't written to the file.. I did it manually now..

Reply:
Problem was: entity type creator has got a bug, the checkbox in the UI is mislinked to visibleByDefault